libs/langchain/langchain/chains/openai_functions/base.py

```diff
@@ -426,14 +231,3 @@ class _OutputFormatter(BaseModel):
         output_parser=output_parser,
         **kwargs,
     )
-
-
-__all__ = [
```

Comment:
if someone importing from here, this is backwards breaking

Reply:
i just moved it to the top, still in file